packages/strapi-design-system/src/InputJSON/decorationExtension.js

```diff
@@ -0,0 +1,38 @@
+import { StateField, StateEffect } from '@codemirror/state';
+import { EditorView, Decoration } from '@codemirror/view';
+
+// Effects can be attached to transactions to communicate with the extension
+const addMarks = StateEffect.define();
+const filterMarks = StateEffect.define();
+
+const lineHighlightMark = Decoration.mark({
+  attributes: { style: 'background-color: yellow; color: black' },
+});
```

Comment:
Now that we move this component into the DS: did you check these colors with Maeva? I feel like we could improve these a lot ...